packages/eslint-plugin/docs/rules/no-explicit-any.md

```diff
@@ -6,8 +6,8 @@ description: 'Disallow the `any` type.'
 >
 > See **https://typescript-eslint.io/rules/no-explicit-any** for documentation.
 
-The `any` type in TypeScript is a dangerous "escape hatch" from the type system.
-Using `any` disables many type checking rules and is generally best used only as a last resort or when prototyping code.
+The `any` type in TypeScript is a potentially dangerous "escape hatch" from the type system.
+Using `any` disables many type checking rules and is generally best used only when it's required or when prototyping code.
```

Comment:
> when it's required

-1 from me on this change: "required" is, at best, just as vague as "last resort". While "last resort" both conveys our disdain for `any` (good) and indicates you should have tried many things first (also good).

Reply:
I can get on-board with that as long as it's clear that `any` is not always an error that needs fixing.